package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@biconomy/account-contracts",
+  "name": "@bcnmy/scw-contracts",
```

Comment:
Don't think we should change it right now.. need some inputs from @livingrockrises here.

Reply:
we don't have any organisation on npm named 'bcnmy'. If anything we can rename on github from bcnmy -> biconomy. but on npm imo these should start with @biconomy... only

Reply:
I named it like the repo but you're right. Should we use the same name everywhere to have better consistency? How about adding `account-abstraction` to the name? Like `biconomy-account-abstraction`. What do you think?

Reply:
For better discoverability we can use
@biconomy/account-abstraction

Reply:
no. we are talking about scw contracts package name or organisation name?

Reply:
Adam suggested org name and @tomarsachin2271 you suggestion is mix up for a package name for account contracts. @biconomy/smart-account or @biconomy/aa-smart-account will also have good visibility.

Reply:
I will keep it unchanged for now, we'll discuss it later